### ISSUE TITLE
Reproduced result on imagenet is lower than paper

### ISSUE THREAD

**Issue by courao** [youngkyunJang/Deep-Hash-Distillation#3]
Hi, thank you for awesome project. 
I've tried your experiment setting, Max map produced by the model is lower than the value in the paper,
eg. ImageNet AlexNet bit 64 is 0.716 vs 0.721 

I tried several times and still cannot get higher map, do you have any other specific hyper-parameter settings?

**Comment by youngkyunJang** [comment]
For AlexNet example, maybe employ batch norm at the end of network could improve some performance. I'm preparing for new updates in this code, and will be released soon.
